BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java

```diff
@@ -114,15 +129,17 @@ public void showRecentTransactions(List<? extends Transaction> transactionList)
             if (transactionList.get(i).getValue() != null && transactionList.get(i).getCurrency() != null) {
                 details.add(transactionList.get(i).getValue().toString() + " " + transactionList.get(i).getCurrency().toString());
             }
-            if (transactionList.get(i).getDirection() != null) {
-                details.add("" + transactionList.get(i).getDirection().toString());
+            if (transactionList.get(i).getValue() < 0) {
+                details.add("OUT");
+            } else {
+                details.add("IN");
```

Comment:
If a transaction list has a direction, why not `details.add(transactionList.get(i).getDirection())`?